.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: php
 
 php:
-  - '7.1'
```

Comment:
There was a bug in psalm 4.5.0 and 4.5.1 and there was a parse error on PHP 7.1 and 7.2 that maybe triggered this change.
Now latest psalm version 4.5.2 should run fine.

Anyway, I'm good with removing support for 7.1 and 7.2. 👍🏻

Reply:
Even then, I removed it because I bumped to `phpunit/phpunit:^9.5`, so testing on EOL'd PHP versions is not viable anyway.